.env.example

```diff
@@ -28,3 +28,7 @@ EXPENSIFY_ACCOUNT_ID_RECEIPTS=-1
 EXPENSIFY_ACCOUNT_ID_REWARDS=-1
 EXPENSIFY_ACCOUNT_ID_STUDENT_AMBASSADOR=-1
 EXPENSIFY_ACCOUNT_ID_SVFG=-1
+
+FB_API_KEY=YOUR_API_KEY
+FB_APP_ID=YOUR_APP_ID
+FB_PROJECT_ID=YOUR_PROJECT_ID
```

Comment:
Can you add a new empty line at the end of the file please? This will help avoid lint warnings

Reply:
Also are these public API keys?